product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx

```diff
@@ -12,7 +12,7 @@ legacyRedirectsGenerated:
 You can assess an Oracle database schema for compatibility with EDB Postgres Advanced Server. You can also generate a report about the assessment.
 
 !!!note 
-   Review the [release notes](../01_mp_release_notes/) and [known issues](../known_issues_notes.mdx) section for an overview of unsupported schema objects.
+   Review the [release notes](../01_mp_release_notes/) and [known issues](../known_issues_notes.mdx) for an overview of unsupported schema objects.
```

Comment:
@gvasquezvargas  I have a query in the next procedure. Why is there a screen capture of the New Project dialog box? We don't include screen captures of dialog boxes unless they are filled in in a certain way that is instructive.

Reply:
I am not sure, as the screenshots are before my time. If you believe we should rethink how we use screenshots on this page, let's sync and we can pitch something to the PM/team.

Reply:
It's not really something we need to pitch. We don't include dialog boxes as a rule. It looks like you added it a few months ago. I'm going to remove it.